### ISSUE TITLE
A few enhancement ideas

### ISSUE THREAD

**Issue by ThePrudentNinja** [dignajar/bludit#374]
While writing a blog post, I noticed a couple of possible improvements.

One is the ability to save drafts. It would be nice to be able to have a "save draft" button that could let you essentially take breaks while writing long posts and come back to it later right in the blogging platform. Sure, you could write it in an office document editor, but due to formatting differences, formatting won't copy over and that means more work. Being able to save a draft right in Bludit and pick it up later will prevent this extra work and make it more pleasurable.

The next idea is to have scrolling on the post editor text box itself so that the little button bar thing above it is always on the screen. When doing formatting, it can be tedious to have to highlight the text, then scroll back up, then click the button, then scroll back down. If the text box itself had a scrollbar, this would be a non-issue.

One more enhancement idea is with the "maintenance" plugin. When activated, even the admin can't access the site. If the admin could access the site, this would allow the admin to have the site down while doing things like theme modifications or other site changes that they don't want a viewer to accidentally see. For SEO, it might be best to return a 503 error code when in maintenance mode. This suggestion with this plugin really isn't that important as it only affects admins when they first install Bludit and maybe on rare occasion. It would be an improvement though

The final enhancement idea is to do better with the plugin page here: plugins.bludit.com
It is not clear enough whether the plugin supports the version of Bludit you have installed and lacks any sort of thumbnails. I've "installed" a number of plugins only to be greeted with "This plugin is incompatible with Bludit v1.5.2" many times. This improvement would make it easier for new users to find compatible plugins for their Bludit with less time wasted.

This is about all I can think of at this time.

**Comment by acrox999** [comment]
(1) It's already there. See the right sidebar when editing a post. Change
Published to Draft.

(2) Good idea but I guess this depends on the editor plugin that you're
using itself.

(3) I think that's because no one has any powers outside the control panel
thus even admins are restricted. Maybe check for cookies?

(4) I agree. Even the Themes website doesn't show any previews.

On Nov 25, 2016 10:17, "ThePrudentNinja" <notifications@github.com> wrote:

While writing a blog post, I noticed a couple of possible improvements.

One is the ability to save drafts. It would be nice to be able to have a
"save draft" button that could let you essentially take breaks while
writing long posts and come back to it later right in the blogging
platform. Sure, you could write it in an office document editor, but due to
formatting differences, formatting won't copy over and that means more
work. Being able to save a draft right in Bludit and pick it up later will
prevent this extra work and make it more pleasurable.

The next idea is to have scrolling on the post editor text box itself so
that the little button bar thing above it is always on the screen. When
doing formatting, it can be tedious to have to highlight the text, then
scroll back up, then click the button, then scroll back down. If the text
box itself had a scrollbar, this would be a non-issue.

One more enhancement idea is with the "maintenance" plugin. When activated,
even the admin can't access the site. If the admin could access the site,
this would allow the admin to have the site down while doing things like
theme modifications or other site changes that they don't want a viewer to
accidentally see. For SEO, it might be best to return a 503 error code when
in maintenance mode. This suggestion with this plugin really isn't that
important as it only affects admins when they first install Bludit and
maybe on rare occasion. It would be an improvement though

The final enhancement idea is to do better with the plugin page here:
plugins.bludit.com
It is not clear enough whether the plugin supports the version of Bludit
you have installed and lacks any sort of thumbnails. I've "installed" a
number of plugins only to be greeted with "This plugin is incompatible with
Bludit v1.5.2" many times. This improvement would make it easier for new
users to find compatible plugins for their Bludit with less time wasted.

This is about all I can think of at this time.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/dignajar/bludit/issues/374>, or mute the thread

**Comment by ThePrudentNinja** [comment]
I didn't notice there was a dropdown menu. I only noticed because you said the drafts thing was there and I didn't see it, so started clicking things. Might want to improve that aspect lol.

**Comment by clickwork-git** [comment]
**Maintenance**

Bludit is open source. Contributions to the code are welcome. ;-) 

**Plugins and Themes**

An issue (enhancement) is already opened at:

https://github.com/dignajar/bludit/issues/360

Also there is an installation where all available themes are installed:

http://themes.bludit.ch

**Comment by Turqueso** [comment]
about what @ThePrudentNinja, I've said it before, a dropdown menu it's not always obvious. An accordion would be more clear since it doesn't hide option behind clicks.

**Comment by clickwork-git** [comment]
Next time: Can you please discuss a bunch of ideas in the forum or open several "issues". Otherwise it's not possible to assign an issue to a label and close it, when it is done.

**Comment by ThePrudentNinja** [comment]
Didn't realize there was a forum, will keep that in mind.

**Comment by Turqueso** [comment]
is there a forum? also we have gitter, shouldn't this link be on the README?

**Comment by clickwork-git** [comment]
If you install Bludit the first post (with the link to admin) informs about the forum, Gitter, Facebook and Twitter. ;-)

Also the website of Bludit with the download link etc. has a link to the forum.

**Comment by ThePrudentNinja** [comment]
"If you install Bludit the first post (with the link to admin) informs about the forum, Gitter, Facebook and Twitter. ;-)

Also the website of Bludit with the download link etc. has a link to the forum."

Not sure how common it is, but I never read the "sample post" of any blogging platform or the example data of any software for that matter. I noticed the forum after it was mentioned by looking at the very bottom of the bludit page. I admit I overlooked it lol. Will keep it in mind for future suggestions.
